CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+[npm][1]
+
+[1]: https://www.npmjs.com/package/@google-cloud/vision
+
+## v0.21.0
+
+### Implementation Changes
+
+- Re-generate library using /synth.py (#105)
```

Comment:
@theacodes it occurs to me that we could have many of these, and that they say little to the user about what actually changed.  A few questions:
- How did you decide if this was a semver major/minor/patch release?  
- Should we aim for more human descriptions here?
- Should we consider lumping anything with this specific commit title together, and showing a single line item with all the PR#s?

Reply:
I put minimal effort into this. Y'all editorialize your release notes however you want. That's kind of the point of this PR - to give human review to the releasenotes.

Reply:
It would have be idiomatic to have the first section be the 'Notable changes'. These are implementation details, and probably don't need to mentioned other than in a '# Other changes' section.

Other tools that exist in the Node.js ecosystem (e.g. [release](https://www.npmjs.com/package/release)) do an initial draft of the notes but let a human edit them. Using conventional commit standard gets rid of a lot of subjectivity at the time of releasing, but highlighting what is notable still requires a human.